XMonad/Util/Brightness.hs

```diff
@@ -0,0 +1,65 @@
+{-# LANGUAGE ScopedTypeVariables #-}
+
+-- | Module to control the brightness of the screen in linux environments
+module XMonad.Util.Brightness
+    ( increase
+    , decrease
+    , change
+    ) where
+
+import XMonad
+import Data.Traversable (traverse)
+import System.IO (hPutStrLn, stderr)
+import Data.Bitraversable (bitraverse)
+import Control.Monad (join)
+import Data.Bifunctor (first)
+import Control.Exception (try)
+import Control.Applicative (liftA2)
+import Data.ByteString.Char8 (unpack)
+import qualified Data.ByteString as BS
+
+maxfile = "/sys/class/backlight/intel_backlight/max_brightness"
+currentfile = "/sys/class/backlight/intel_backlight/brightness"
```

Comment:
Can you document both these files as part of Haddock documentation and mention that they need to be present for it to work. Also can you mention the permission issue and the method to change it properly (via groups IIRC) in the Haddocks ?

Reply:
By group were you reffering to group of function documentation?
I thought of putting the doc about permission issues in the module doc. What do you think?

Reply:
@giovanifss Yeah, module doc would be a good place. Have a separate section for it and add the doc there. Something like this: https://www.stackage.org/haddock/lts-11.2/persistent-2.8.1/Database-Persist.html#g:1

Reply:
Everything is fine except for the docs for maxfile and currentfile. I couldn't find a way to document these functions without exporting them besides creating an Internal module and putting it there. Do you have any suggestions?

Reply:
Just document in the haddock that you are assuming those two files to be present in the system.

Reply:
Yes, I know. The problem is: when I document those functions, haddock does not add them to the final documentation because they are not exported. Exporting those functions does not seem to be a good idea, so I asked if you know a way to make haddock generate the docs without exporting them.

Reply:
I've added as lists because it was cleaner in the generated HTML. If you prefer, I can return to headings (https://www.haskell.org/haddock/doc/html/ch03s08.html#headings).